### PR TITLE
Update FSM_MVP_Specs.yml

### PR DESCRIPTION
**Pull request by CGantert345**
- exchange added
- fulfillment corrected
- non trip offer added

### CODE REVIEW

**Comment by CGantert345** [comment]
enthält auch die Änderungen aus dem vorherigen commit

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 1]
```diff
@@ -830,7 +830,7 @@ paths:
         '501':
           description: Not implemented
           
-  '/offers/{offerId}/admissions/{admissionId}':
+  '/offers/{offerId}/admissions/{admissionsId}':
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:833]: Why plural?

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 2]
```diff
@@ -1069,7 +1070,7 @@ paths:
         '501':
           description: Not implemented
           
-  '/offers/{offerId}/ancillaries/{ancillaryId}':
+  '/offers/{offerId}/ancillaries/{ancilliaryId}':
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:1073]: Why 'ancilliaryId' -  it's spelled ancillary :-)

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 3]
```diff
@@ -1255,7 +1256,23 @@ paths:
           required: false
           description: >-
             End date for the travel date range to search.
-
+        - name: OriginId
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:1259]: lowercase

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 4]
```diff
@@ -1255,7 +1256,23 @@ paths:
...
-
+        - name: OriginId 
+          in: query
+          schema:
+            type: string
+            format: uuid
+            example: d290f1ee-6c54-4b01-90e6-d701748f0851
+          required: false
+          description: departure locationId one of the travels in the booking to search on
+        - name: DestinationId
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:1267]: lowercase

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 5]
```diff
@@ -3802,6 +4483,151 @@ components:
...
+        places:
+          description: list of places included in the layout
+          type: array
+          items:
+            type: object
+            properties:
+              x:
+                description: horizontal coordinate of the center of the place
+                type: number
+              'y':
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:4514]: 'y' --> y

File: IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml [hunk 6]
```diff
@@ -4546,3 +5374,76 @@ components:
       additionalProperties: false      
       
     #Fare-specific elements end
+    
+    OfferSearchDef:
```
Comment (schlpbch) [inline IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml:5378]: Think: Can an offer have references to a trip?

**Pull request merged**